modules/test/playwright/pages/document-library-web/gogoShell.page.ts

```diff
@@ -0,0 +1,35 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+// @ts-ignore
+
+import {Locator, Page} from '@playwright/test';
+
+export class GogoShellPage {
+	readonly commandInput: Locator;
+	readonly page: Page;
+	readonly executeButton: Locator;
+
+	constructor(page: Page) {
+		this.page = page;
+
+		this.commandInput = this.page.getByLabel('Command');
+		this.executeButton = this.page.getByRole('button', {name: 'Execute'});
+	}
+
+	async goto() {
+		await this.page.getByLabel('Open Applications MenuCtrl+').click();
+		await this.page.getByRole('tab', {name: 'Control Panel'}).click();
+		await this.page.getByRole('menuitem', {name: 'Gogo Shell'}).click();
```

Comment:
Looks like a good place to use `ProductMenuPage` here.

Reply:
but `ProductMenuPage` is more for the left side menu. Maybe I should create a new page named `AplicationsMenuPage`?

Reply:
I like more the creation on the AplicationsMenuPage

Reply:
Sorry, you both are right `AplicationsMenuPage ` will work.

Reply:
Changed, I finally used `ApplicationsMenuPage`.